PWGGA/GammaConvBase/AliConversionPhotonCuts.cxx

```diff
@@ -595,6 +597,24 @@ void AliConversionPhotonCuts::InitCutHistograms(TString name, Bool_t preCut){
       //    }
   }
 
+  fHistoTrackPID=new TH2F(Form("TrackPID %s",GetCutNumber().Data()),"TrackPID",15,-0.5,14.5,250,0,25);
```

Comment:
Could you please make this histogram being optional using a switch? We are more on the upside of memory usage, so that adding another 2D histogram should be carefully considered

Reply:
ok sure
i will change this

Reply:
> Could you please make this histogram being optional using a switch? We are more on the upside of memory usage, so that adding another 2D histogram should be carefully considered

Done

Reply:
i will do another commit sorry where i add the switch to the addtask